templates/3/sites/ldap_header.epp

```diff
@@ -0,0 +1,4 @@
+# This file is managed by Puppet.  Changes will be overwritten
+# at the next puppet run.
+#
+server <%= $freeradius::v3::sites::ldap::site_name %> {
```

Comment:
Why the trailing open curly bracket '{' ?

Reply:
It is a header.  The footer has the closing curly bracket.

Reply:
ah, gotcha